Sources/RoktUXHelper/UI/Components/CreativeResponseComponent.swift

```diff
@@ -103,77 +103,85 @@ struct CreativeResponseComponent: View {
+                .alignSelf(alignSelf: flexStyle?.alignSelf,
+                           parent: config.parent,
+                           parentHeight: parentHeight,
+                           parentWidth: parentWidth,
+                           parentVerticalAlignment: parentOverride?.parentVerticalAlignment,
+                           parentHorizontalAlignment: parentOverride?.parentHorizontalAlignment,
+                           applyAlignSelf: true)
             // margin must apply after the touchable area and before readSize
-            .margin(spacing: spacingStyle, applyMargin: true)
-            .readSize(spacing: spacingStyle) { size in
-                availableWidth = size.width
-                availableHeight = size.height
-            }
-            .onTapGesture {
-                handleLink()
-            }
+                .margin(spacing: spacingStyle, applyMargin: true)
+                .readSize(spacing: spacingStyle) { size in
+                    availableWidth = size.width
+                    availableHeight = size.height
+                }
+                .onTapGesture {
+                    handleLink()
+                }
             // consecutive gestures to track when long press is held vs released
-            .gesture(LongPressGesture()
-                .sequenced(before: LongPressGesture(minimumDuration: .infinity))
-                .updating($isPressingDown) { value, state, _ in
-                    switch value {
-                    case .second(true, nil):
-                        state = true
-                    default:
-                        break
+                .gesture(LongPressGesture()
+                    .sequenced(before: LongPressGesture(minimumDuration: .infinity))
+                    .updating($isPressingDown) { value, state, _ in
+                        switch value {
+                        case .second(true, nil):
+                            state = true
+                        default:
+                            break
+                        }
+                    })
+                .onChange(of: isPressingDown) { value in
+                    if !value {
+                        // handle link when long press is released
+                        handleLink()
                     }
-                })
-            .onChange(of: isPressingDown) { value in
-                if !value {
-                    // handle link when long press is released
-                    handleLink()
                 }
-            }
-            .onLongPressGesture(perform: {
-            }, onPressingChanged: { isPressed in
-                self.isPressed = isPressed
-                updateStyleState()
-            })
+                .onLongPressGesture(perform: {
+                }, onPressingChanged: { isPressed in
+                    self.isPressed = isPressed
+                    updateStyleState()
+                })
+        } else {
+            EmptyView()
+        }
```

Comment:
my only comment here is can we make it something like:
```
if isExternalResponseOption {
    EmptyView()
    return
}

... previous code unformatted at all
```

To reduce number of line changes.  Not a big deal, but it's a practice we try to do in our JS repos

Reply:
I tried to do this but if you explicitly add a return here you must add it elsewhere, which nullifies the purpose of this change anyway (reducing the number of line changes). Also you run into added problems with the return type of `some View` and that ends up requiring more code unnecessary code changes.